src/data/directory/lgu/region-i-ilocos-region.json

```
@@ -418,37 +417,37 @@
             "name": "JOEY WARREN A. BRAGADO"
           },
           "vice_mayor": {
-            "name": "HONESTO S. FORONDA"
+            "name": "FERDINAND BONGBONG V. BANUA JR"
           }
         },
         {
           "municipality": "San Esteban",
           "zip_code": "2706",
           "mayor": {
-            "name": "RAY M. ELAYDO II"
+            "name": "MARIA DIVINA GRACIA E. ETALIN"
           },
           "vice_mayor": {
-            "name": "MARIA DIVINA GRACIA E. ETALIN"
+            "name": "RAY M. ELAYDO II"
           }
         },
         {
           "municipality": "San Ildefonso",
           "zip_code": "2728",
           "mayor": {
-            "name": "CHRISTIAN DANIEL A. PURISIMA"
+            "name": "AMANTE PURISIMA II"
           },
           "vice_mayor": {
-            "name": "ROBERT R. RIEGO"
+            "name": "RENATO 'RENI' PECHO"
           }
         },
         {
           "municipality": "San Juan",
           "zip_code": "2731",
           "mayor": {
-            "name": "MA. ELAINE A. SARMIENTO"
+            "name": "BENJAMIN SARMIENTO"
```

```suggestion
            "name": "BENJAMIN V. SARMIENTO"
```

ref: https://lgu201.dilg.gov.ph/view.php?r=01&p=029&m=20